Library/Formula/thewidgetfactory.rb

```diff
@@ -0,0 +1,66 @@
+class Thewidgetfactory < Formula
+  desc "The Widget Factory is a showcase tool for GTK+ widgets"
+  homepage "http://www.stellingwerff.com/?page_id=10"
+  url "http://www.stellingwerff.com/TheWidgetFactory/thewidgetfactory-0.2.1.tar.gz"
+  sha256 "10bbbe50dc23a0cd47762749c6bb402559721951f3c6af40366258593bcb1ce4"
+
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!